### ISSUE TITLE
Support for IP ranges in allowed_sources

### ISSUE THREAD

**Issue by paul-pearce** [mavam/gitdub#6]
Given that github now has entire ranges, this seems necessary.

**Comment by mavam** [comment]
Yep, that's a valid issue. Do you plan to use that feature in your deployment?

**Comment by paul-pearce** [comment]
Yes

**Comment by stas00** [comment]
I second this request. The list of ranges is here under hooks section: https://api.github.com/meta
Thank you!

**Comment by mavam** [comment]
Unfortunately I currently lack the cycles to implement this feature, though I'd be more than happy to review a PR.

**Comment by stas00** [comment]
Both remaining open issues can be closed now.
